docs/generated/errors-list.md

```diff
@@ -109,6 +109,13 @@ The `galasactl` tool can generate the following errors:
 - GAL1106E: Could not get security bearer token from API server. Reason: '{}'. Please ensure you have allocated a personal access token and configured your client program by storing it in your galasactl.properties file together with the related client ID and secret
 - GAL1107E: Could not get security bearer token from file '{}'. Reason: '{}'. Please ensure you are authenticated by running 'galasactl auth login' and that your personal access token has not expired or been revoked
 - GAL1108E: Invalid bearer token. Please ensure you are authenticated by running 'galasactl auth login' and that your personal access token has not expired or been revoked
+- GAL1109E: Unable to access the file. Reason: '{}'
+- GAL1110E: Could not get the content of the yaml file. Reason: '{}'
+- GAL1111E: Error unmarshalling yaml. Invalid yaml. Reason: '{}'. Invalid yaml:'{}'
```

Comment:
Unmarshalling is a bit non-english/too-compsci.

Error parsing the file '{}'. The file is not in the expected yaml format. Reason:'{}'. 

That might be better ?